test/phoenix_live_view/integrations/components_test.exs

```diff
@@ -1,3 +1,4 @@
+# TODO: Rename this file and test case to stateful_component
```

Comment:
Shall we tackle this TODO now?

Reply:
I'd rather do it in a separate PR along with other changes in that regard.